generate-manifest.py

```diff
@@ -76,6 +77,30 @@ def edit_manifest(data, arch, branch, runtime_version):
             data['modules'].insert(0, gtk_module)
             break
 
+def sha256(filename):
+    checksum = hashlib.sha256()
+    with open(filename, 'rb') as file:
+        for data in iter(lambda: file.read(65536), b''):
+            checksum.update(data)
+    return checksum.hexdigest()
+
+def add_fonts_module(data):
+    """Add fonts module to manifest json"""
+
+    sources = [ { 'type': 'file', 'path': 'fonts-makefile', 'dest-filename': 'Makefile' } ]
+
+    for font in os.listdir('fonts'):
+        if font.endswith('.zip'):
+            path = 'fonts/' + font
+            sources.append({ 'type': 'file',
+                             'path': path,
+                             'dest-filename': font,
+                             'sha256': sha256(path)})
+
+    data['modules'].append({ 'name': 'default-theme-fonts',
```

Comment:
I'd rather have `data['modules'].prepend()` here, for two reasons:

 - early failure
 - keep the `os-release` module at the end, as it's the last part of the build process